ecs/fargate/main.tf

```diff
@@ -13,7 +13,7 @@ resource "aws_cloudwatch_log_group" "main" {
 }
 
 # ------------------------------------------------------------------------------
-# IAM
+# IAM - Service role- is this used?
```

Comment:
It does not seem to be used here but it should, we are using it in the regular service module: https://github.com/TeliaSoneraNorge/telia-terraform-modules/blob/master/ecs/service/main.tf#L38
Could you add it here or should we do it in another PR?

Reply:
@mikael-lindstrom regarding iam_role for service; (https://www.terraform.io/docs/providers/aws/r/ecs_service.html)

This parameter is required if you are using a load balancer with your service, but only if your task definition does not use the awsvpc network mode. If using awsvpc network mode, do not specify this role.

We don't need the role since we are using awsvpc networking mode for fargate, if I understand the details correctly.

Reply:
You are correct 🥇 We can remove this, either we do it here or in another PR. Approving so you can merge if we should do this in another PR.